File: Javascript/Exercicio03.js
const prompt = require('prompt-sync')();

var b = prompt("Digite um Número para a Base: ");
var e = prompt("Digite um Número para o Expoente: ");
var i = 0;
var x = 1;

if (e == 0) {
    x = 1;
} else {
   do{
        i++;
        x = x*b;
    }	while(i < e); 
}

console.log(b+" Elevado à "+ e + " = "  + x );

File: Java/Exercicio03.java
package Java;

import java.util.Scanner;

public class Exercicio03 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite um Número para a Base: ");
        int b = scanner.nextInt();

        System.out.println("Digite um Número para o Expoente: ");
        int e = scanner.nextInt();
		int i = 0;
		int x = 1;
        if (e == 0) {
            x = 1;
        } else {
           do{
                i++;
                x = x*b;
            }	while(i < e); 
        }
		System.out.println(b+" Elevado à "+ e + " = "  + x );
    }
}
